mssqlcli/mssql_cli.py

```diff
@@ -243,7 +244,9 @@ def initialize_logging(self):
         if log_level.upper() == 'NONE':
             handler = logging.NullHandler()
         else:
-            handler = logging.FileHandler(os.path.expanduser(log_file), encoding='utf-8')
+            # creates a log buffer with max size of 100 MB and one backup file
+            handler = RotatingFileHandler(os.path.expanduser(log_file), encoding='utf-8',
```

Comment:
A 100MB file is still pretty large. Lets chunk the files into a size that someone can forward. How about 5 files with a max size of 20MB? Or 10 files 10MB?

Reply:
Great idea, will change to 5 files/20 MB.